<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NUi::CTransformationSettings</class>
 <widget class="QWidget" name="NMediaManager::NUi::CTransformationSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="treatAsTVShowByDefault">
     <property name="text">
      <string>Treat as TV Show by default? (when auto determine fails)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exactMatchesOnly">
     <property name="text">
      <string>Exact Matches Only?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="verifyMediaTags">
     <property name="title">
      <string>Verify Media Tags?</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="verifyMediaTitle">
        <property name="text">
         <string>Verify Media Title is not empty or incorrect?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="verifyMediaDate">
        <property name="text">
         <string>Verify Media Date is not empty or incorrect?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
